beestation.dme

```diff
@@ -269,7 +269,6 @@
 #include "code\controllers\subsystem\parallax.dm"
 #include "code\controllers\subsystem\pathfinder.dm"
 #include "code\controllers\subsystem\persistence.dm"
-#include "code\controllers\subsystem\ping.dm"
```

Comment:
The hell is going on with this file?

Reply:
That file shouldn't even exist on the repo